src/services/authentication/authentication.context.js

```diff
@@ -23,6 +23,24 @@ export const AuthenticationContextProvider = ({ children }) => {
       });
   };
 
+  const onRegister = (email, password, repeatedPassword) => {
+    if (password !== repeatedPassword) {
+      setError("Error: Passwords do not match");
+      return;
+    }
+    firebase
```

Comment:
If you are on firebase 9 please checkout master for a more up to date reference:
https://github.com/mobinni/MealsToGo/blob/master/src/services/authentication/authentication.service.js